### ISSUE TITLE
OutdatedModelError after restart the server

### ISSUE THREAD

**Issue by YinanDanielZhou** [rvinzent/django-dynamic-models#26]
Hello @rvinzent , we are using your code to create dynamic models. However, after we shutdown the django server and restart using `python3 manage.py runserver 0.0.0.0:8000` , we would no longer be able to write new rows into the dynamic models we created before the restart. More specifically, when we call `annotations_m.objects.create(**kwargs)` (`annotations_m` is a model from calling `.as_model()`), we get a `OutdatedModelError` from `check_model_schema` in `dynamic_models/factory.py`. 
Before we restart the server, we do not get such Exception when calling the '.create(**kwargs)` function.
Could you please help us out?

**Comment by rvinzent** [comment]
@YinanDanielZhou what are you using for Django `cache`? If the cache is still alive while you restart the server then I think it will become invalid. I am planning on re-implementing this cache and "outdated model" check whenever I find the time/motivation.

Please don't hesitate to fork the repo and submit a PR if you can find a fix 🙂

**Comment by YinanDanielZhou** [comment]
There is no specifications about `cache` in the `settings.py` so I am guessing it is using the default cache. Could you please explain to me more what this `check_model_schema` does? I don't really understand `Called on pre_save to guard against the possibility of a model schema change
    between instance instantiation and record save.` I will try to fix it myself.

**Comment by YinanDanielZhou** [comment]
hey @rvinzent , I was not able to find a good fix to it and I am thinking of just ignoring the `OutdatedModelError`. What kind of danger would that cause?

**Comment by rvinzent** [comment]
@YinanDanielZhou it would help me out a lot of you are able to provide some code with a reproduction if the bug. 

The error is intended to prevent the case when a model is generated, and then the schema is updated, so the model shouldn't be used anymore and a new one should be generated with `as_model()`. 

If you ignore it you might see some other database errors but I don't think anything with ruin your data.

**Comment by ric-sapasap** [comment]
Hi, I'm also experiencing the same problem.
For reference, I'm using [django rest framework](django-rest-framework.org/) to dynamically create CRUD endpoints for the dynamic schemas

```
    def get(self, request: Request, *args, **kwargs):
        slug_schema = ModelSchema.objects.get(name="loans")
        SlugModel = slug_schema.as_model()
        instance = SlugModel.objects.create(**{
            "name": "TEST NAME",
            "description": "TEST DESCRIPTION"
        })
        print(instance.id, flush=True)
        return Response(status=HTTPStatus.OK)
```

The `ModelSchema` loans is loaded into the database via django fixtures and migrations.
When I run the above, I get the following error:
![image](https://user-images.githubusercontent.com/52158902/93132324-042bcd80-f708-11ea-9506-bcc91c87d61e.png)

I also have pytest test cases for this using an SQLite database and it works flawlessly there.

Any pointers on how to fix this?

**Comment by rvinzent** [comment]
I believe I've found the issue and fixed in in #27 and #28.

The fix has been released on version `0.2.1rc0`.

@ric-sapasap @YinanDanielZhou would either of you mind confirming that the fix works?

Thanks!

**Comment by ric-sapasap** [comment]
Copy on this @rvinzent. Will test the soonest. Thank you! :+1:

**Comment by ric-sapasap** [comment]
Hi, just tested this. It works for my use case. Thank you so much @rvinzent!

**Comment by rvinzent** [comment]
@ric-sapasap  Great! I re-released this fix as `0.3.0` because I removed some public methods and added a migration to remove a private field `_modified` on the `ModelSchema` model. 

I'll go ahead and update the wiki. Thanks for testing!
